README.md

```diff
@@ -2,75 +2,111 @@
-
-| stac-server Version | STAC Version  |
-| -------- | ---------- |
-| 0.1.x    | 0.9.x      |
-| 0.2.x    | <1.0.0-rc.1 |
-| 0.3.x    | 1.0.0 |
+- [stac-server](#stac-server)
+  - [Overview](#overview)
+  - [Migration](#migration)
+    - [0.3 -> 0.4](#03---04)
+  - [Usage](#usage)
+  - [Deployment](#deployment)
+  - [Ingesting Data](#ingesting-data)
+    - [Subscribing to SNS Topics](#subscribing-to-sns-topics)
+    - [Ingest Errors](#ingest-errors)
+  - [Development](#development)
+    - [Running Locally using AWS Resources](#running-locally-using-aws-resources)
+    - [Running Locally using Local Resources](#running-locally-using-local-resources)
+  - [About](#about)
+
+## Overview
+
+Stac-server is an implementation of the [STAC API specification](https://github.com/radiantearth/stac-api-spec) for searching and serving metadata for geospatial data, including but not limited to satellite imagery). The STAC and STAC API versions supported by a given version of stac-server are shown in the table below. Additional information can be found in the [CHANGELOG](CHANGELOG.md)
+
+| stac-server Version | STAC Version | STAC API Version |
+| ------------------- | ------------ | ---------------- |
+| 0.1.x               | 0.9.x        | 0.9.x            |
+| 0.2.x               | <1.0.0-rc.1  | 0.9.x            |
+| 0.3.x               | 1.0.0        | 1.0.0-beta.2     |
+| 0.4.x               | 1.0.0        | 1.0.0-beta.5     |
 
 The following APIs are deployed instances of stac-server:
 
-| Name     | Version   | Description |
-| -------- | ----      | ----        |
-| [Earth Search](https://earth-search.aws.element84.com/v0/) | 1.0.0-beta.2 | Catalog of some AWS Public Datasets |
+| Name                                                       | STAC Version | STAC API Version | Description                         |
+| ---------------------------------------------------------- | ------------ | ---------------- |
+| [Earth Search](https://earth-search.aws.element84.com/v0/) | 1.0.0-beta.2 | 0.9.0            | Catalog of some AWS Public Datasets |
+| [Landsat Look](https://landsatlook.usgs.gov/stac-server)   | 1.0.0        | 0.9.0            |                                     |
+
+## Migration
+
+### 0.3 -> 0.4
 
+The Serverless Framework supports provisioning AWS resources, but it does not support updating existing resources. In 0.4, the default Elasticsearch version has been updated from 7.9 to 7.10. Continuing to use 7.9 should not cause any problems, but it recommended that you manually upgrade to 7.10 by going to [AWS Console - Amazon OpenSearch Service](https://console.aws.amazon.com/esv3/home), choosing the Elasticsearch domain used by your stac-server deployment (e.g., stac-server-{stage}-es), choose Upgrade from the Actions menu, and then upgrade to Elasticsearch 7.10.
```

Comment:
the actual update for 7.10 is going to be in another PR, but I was working on it at the same time, and put it in the README along with all of these other changes.